app/services/efile/gyr_efiler_service.rb

```diff
@@ -3,6 +3,11 @@ module Efile
   class GyrEfilerService
     CURRENT_VERSION = 'f5eeb816f6c919fff8d5c742062e664b1f4cd13a'
     POSTGRES_LOCK_PREFIX = 1640661264
+    RETRYABLE_LOG_CONTENTS = [
```

Comment:
As this list grows, my suggestion would be to create a .yml file for this and load it in! (See: the way we create an array for timezone overrides and/or the way we load in zip codes.